common/build/eslint-config-fluid/minimal-deprecated.js

```diff
@@ -24,14 +24,38 @@ const permittedImports = [
 	"*/index.js",
 ];
 
+// Restricted import patterns for all code.
+const restrictedImportPaths = [
+	// Prefer strict assertions
+	// See: <https://nodejs.org/api/assert.html#strict-assertion-mode>
+	{
+		name: "assert",
+		importNames: ["default"],
+		message: 'Use `strict` instead. E.g. `import { strict as assert } from "assert";`',
```

Comment:
I think all our code can use node:assert now, and its the preferred option so might as well recommend it for both cases even if we don't ban the other one.
```suggestion
		message: 'Use `strict` instead. E.g. `import { strict as assert } from "node:assert";`',
```